pytradfri/smart_task.py

```diff
@@ -216,14 +216,14 @@ def calibrate_time(self) -> Command[None]:
 
         def process_result(result: TypeRaw) -> None:
             gateway_info: GatewayInfo = GatewayInfo(result)
-            if gateway_info.current_time:
-                d_now: dt = gateway_info.current_time
-                d_utcnow: dt = dt.utcnow()
-                diff: timedelta = d_now - d_utcnow
+            if not gateway_info.current_time:
+                return
 
-                self._task.delta_time_gateway_local = diff
+            d_now: dt = gateway_info.current_time
```

Comment:
Do we need these type annotations? Normally mypy will be able to read the types from the objects. It's only when the object is unknown or generic we may need to help mypy and define the type explicitly.

Reply:
Looks like no, I'll remove them.